test/ApproveTest.t.sol

```diff
@@ -0,0 +1,13 @@
+/// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./ZeroState.t.sol";
+
+contract ApproveTest is ZeroState {
+
+    function testApproval() public {
```

Comment:
you don't need to test this basic functionality of the erc20 that you are importing -- assume its already tested

Reply:
I had put this in here cause I couldn't get approval to work properly cause I was trying to call it via the interface in the vault.